### PR TITLE
Aerospike 4.3.0.7

### PR DESCRIPTION
**Pull request by volmarl**
https://www.aerospike.com/download/server/notes.html#4.3.0.7

### CODE REVIEW

File: library/aerospike [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # maintainer: Lucien Volmar <lucien@aerospike.com> (@volmarl)
 
 3.13.0.11: git://github.com/aerospike/aerospike-server.docker@29be954f3d764ae15f0666c39fc14b55f13f1033
-3.16.0.6: git://github.com/aerospike/aerospike-server.docker@961f91929117b71975382ae795c54ea19426bc47
-4.3.0.6: git://github.com/aerospike/aerospike-server.docker@e0a3ec717e07636b4aa032f31fe1aed62ef6039b
-latest: git://github.com/aerospike/aerospike-server.docker@e0a3ec717e07636b4aa032f31fe1aed62ef6039b
+4.1.0.6: git://github.com/aerospike/aerospike-server.docker@cbc02a818af8ea2afb893093d062339a8b851e6d
```
Comment (yosifkit) [inline library/aerospike]: This commit does not seem to exist, did you forget to push it?

Comment (volmarl) [reply library/aerospike]: These two seems to exist:

https://github.com/aerospike/aerospike-server.docker/commit/cbc02a818af8ea2afb893093d062339a8b851e6d

https://github.com/aerospike/aerospike-server.docker/commit/d2fa950e7cb5c10ccc39d778b2aef233440a855a

**Comment by yosifkit** [comment]
warning: insecure protocol git:// detected: git://github.com/aerospike/aerospike-server.docker
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index 01a4cac..e064b8e 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,3 +1,3 @@
 aerospike:3.13.0.11 @ amd64
-aerospike:3.16.0.6 @ amd64
+aerospike:4.1.0.6 @ amd64
 aerospike:latest @ amd64
diff --git a/_bashbrew-list b/_bashbrew-list
index 83c26f0..c142861 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -1,4 +1,4 @@
 aerospike:3.13.0.11
-aerospike:3.16.0.6
-aerospike:4.3.0.6
+aerospike:4.1.0.6
+aerospike:4.3.0.7
 aerospike:latest
diff --git a/aerospike_3.16.0.6/Dockerfile b/aerospike_4.1.0.6/Dockerfile
similarity index 73%
rename from aerospike_3.16.0.6/Dockerfile
rename to aerospike_4.1.0.6/Dockerfile
index 3762c27..c966d76 100644
--- a/aerospike_3.16.0.6/Dockerfile
+++ b/aerospike_4.1.0.6/Dockerfile
@@ -4,19 +4,20 @@
 # http://github.com/aerospike/aerospike-server.docker
 #
 
-FROM ubuntu:xenial
+FROM debian:9.5-slim 
 
-ENV AEROSPIKE_VERSION 3.16.0.6
-ENV AEROSPIKE_SHA256 5f0ed87ac53d5247bfa966a6c9bd769d4398431a1e0425f7c8daac1c47ac84c4
+ENV AEROSPIKE_VERSION 4.1.0.6
+ENV AEROSPIKE_SHA256 98e6d3a4e95bd8119b1d8b3b353a2b06ededd8ee9409401c2a6fc827881b4055
 
 
 # Install Aerospike Server and Tools
 
 
+
 RUN \
   apt-get update -y \
-  && apt-get install -y wget python python-argparse python-bcrypt python-openssl logrotate net-tools iproute2 iputils-ping gettext-base\
-  && wget "https://www.aerospike.com/artifacts/aerospike-server-community/${AEROSPIKE_VERSION}/aerospike-server-community-${AEROSPIKE_VERSION}-ubuntu16.04.tgz" -O aerospike-server.tgz \
+  && apt-get install -y wget python lua5.2 gettext-base \
+  && wget "https://www.aerospike.com/artifacts/aerospike-server-community/${AEROSPIKE_VERSION}/aerospike-server-community-${AEROSPIKE_VERSION}-debian9.tgz" -O aerospike-server.tgz \
   && echo "$AEROSPIKE_SHA256 *aerospike-server.tgz" | sha256sum -c - \
   && mkdir aerospike \
   && tar xzf aerospike-server.tgz --strip-components=1 -C aerospike \
@@ -25,9 +26,13 @@ RUN \
   && mkdir -p /var/log/aerospike/ \
   && mkdir -p /var/run/aerospike/ \
   && rm -rf aerospike-server.tgz aerospike /var/lib/apt/lists/* \
-  && dpkg -r wget ca-certificates \
-  && dpkg --purge wget ca-certificates \
-  && apt-get purge -y
+  && rm -rf /opt/aerospike/lib/java \
+  && dpkg -r wget ca-certificates openssl xz-utils\
+  && dpkg --purge wget ca-certificates openssl xz-utils\
+  && apt-get purge -y \
+  && apt autoremove -y 
+
+  
 
 
 # Add the Aerospike configuration specific to this dockerfile
diff --git a/aerospike_3.16.0.6/Dockerfile.md b/aerospike_4.1.0.6/Dockerfile.md
similarity index 100%
rename from aerospike_3.16.0.6/Dockerfile.md
rename to aerospike_4.1.0.6/Dockerfile.md
diff --git a/aerospike_3.16.0.6/aerospike.template.conf b/aerospike_4.1.0.6/aerospike.template.conf
similarity index 100%
rename from aerospike_3.16.0.6/aerospike.template.conf
rename to aerospike_4.1.0.6/aerospike.template.conf
diff --git a/aerospike_3.16.0.6/entrypoint.sh b/aerospike_4.1.0.6/entrypoint.sh
similarity index 100%
rename from aerospike_3.16.0.6/entrypoint.sh
rename to aerospike_4.1.0.6/entrypoint.sh
diff --git a/aerospike_latest/Dockerfile b/aerospike_latest/Dockerfile
index de386f2..8e287ef 100644
--- a/aerospike_latest/Dockerfile
+++ b/aerospike_latest/Dockerfile
@@ -6,8 +6,8 @@
 
 FROM debian:9.5-slim 
 
-ENV AEROSPIKE_VERSION 4.3.0.6
-ENV AEROSPIKE_SHA256 e8f898211a5fd01c14da8ae1f71468d26cb7d7bac04d9f4674ee61383e8f5de6
+ENV AEROSPIKE_VERSION 4.3.0.7
+ENV AEROSPIKE_SHA256 7586e21fc9c53a152cdd58fb39e9d073ebc71d2898f6e134ae3b1b2deae44022
 
 
 # Install Aerospike Server and Tools
@@ -26,7 +26,6 @@ RUN \
   && mkdir -p /var/log/aerospike/ \
   && mkdir -p /var/run/aerospike/ \
   && rm -rf aerospike-server.tgz aerospike /var/lib/apt/lists/* \
-  && rm -rf /opt/aerospike/lib/java \
   && dpkg -r wget ca-certificates openssl xz-utils\
   && dpkg --purge wget ca-certificates openssl xz-utils\
   && apt-get purge -y \
```

</details>

**Comment by yosifkit** [comment]
Build test of #4832; 2d8259ca; `amd64` (`aerospike`):

```console

$ bashbrew build aerospike:3.13.0.11
Using bashbrew/cache:3bfde95c6e6a467b127c94f2b1ddb078cd6bcefa092facfa876c6bfe90f6674c (aerospike:3.13.0.11)
Tagging aerospike:3.13.0.11

$ test/run.sh aerospike:3.13.0.11
testing aerospike:3.13.0.11
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build aerospike:4.1.0.6
Building bashbrew/cache:7c7aaef6c86cdb484d7e5caf7698d0af24df7a0dd4b486c7146a7a83c7974f02 (aerospike:4.1.0.6)
Tagging aerospike:4.1.0.6

$ test/run.sh aerospike:4.1.0.6
testing aerospike:4.1.0.6
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build aerospike:4.3.0.7
Building bashbrew/cache:02d787e466c6e6340e3cfb50544fddf608722b6a0ea2ab123fd81d89d57b79f7 (aerospike:4.3.0.7)
Tagging aerospike:4.3.0.7
Tagging aerospike:latest

$ test/run.sh aerospike:4.3.0.7
testing aerospike:4.3.0.7
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**